### DISCUSSION TITLE
Saving Spacy's text categorizer model each and every 100th iteration

### DISCUSSION THREAD

**Discussion by 111kannan** [explosion/spaCy#10253] [Help: Other Questions] [answered]
I want to save the models each and every 100th iteration during training. Can anyone suggest to me where I should change the configuration in config.cfg file?

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @111kannan , we don't have a feature for this. The training process only saves the best and last models. If you wish, you might try working around the [`nlp.to_disk`](https://spacy.io/api/language#to_disk) function and write a hook for it, but it's more of a hack.
